src/chartLibraries/easyPie/index.js

```diff
@@ -115,7 +115,7 @@ export default (sdk, chart) => {
     }
 
     const { result } = chart.getPayload()
-    const row = hoverX ? chart.getClosestRow(hoverX[0]) : result.data.length - 1
+    const row = hoverX ? chart.getClosestRow(hoverX[0]) : (result.data?.length || 0) - 1
 
     const rowData = result.data[row]
```

Comment:
By passing the above this will produce an error. Maybe just early return if `!result.data?.length`

Reply:
Updated! 😄